src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1792,8 +1792,7 @@ public SVNAuthentication createSVNAuthentication(String kind) {
 
             @Override
             public StandardCredentials toCredentials(String description) {
-                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName,
-                        getPassword());
+                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName, getPassword());
```

Comment:
@oleg-nenashev It is not a refactoring, I removed only an unnecessary *carriage return*. In my opinion, is more legible.